libkbfs/test_common.go

```diff
@@ -445,24 +445,32 @@ func DisableCRForTesting(config Config, folderBranch FolderBranch) error {
 	return nil
 }
 
-// RestartCRForTesting re-enables conflict resolution for the given
-// folder.
-func RestartCRForTesting(config Config, folderBranch FolderBranch) error {
+// RestartCRForTestingWithCtxMaker re-enables conflict resolution for
+// the given folder.
+func RestartCRForTestingWithCtxMaker(config Config, folderBranch FolderBranch,
```

Comment:
If you go with using a context instead of a context-returning function, then probably best to make it the first argument here, make Restart not take nil, etc.